go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/konflux-ci/multi-platform-controller
 go 1.20
 
 require (
+	github.com/IBM-Cloud/power-go-client v1.6.0
```

Comment:
Question so I can learn: if we use go modules, why are we adding the vendored dependencies to git?

Reply:
thats builds faster and safer, but adds a lot of pain in ___ if you try to add something (like this PR)
maybe @ifireball  knows the historical background....

Reply:
thanks for the explanation

Reply:
I'd drop that, but not a priority ATM.